<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_Path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MicaSenseLive5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Near Infrared  LiveView</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CaptureMode">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Timer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Signal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Single</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Button_PathOpen">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LiveViewOrNo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>网页预览:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_KBH">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>140</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>抗饱和:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_MicaSenseIPSet">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MicaSenseLive2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Green LiveView</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MicaSenseLive4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Red Edge LiveView</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_PathEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>路径修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MicasenseIP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_CaptureMode">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>采集模式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_KBH">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_MicaSenseLive1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string> Blue LiveView</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_SavetoSD">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Button_ClearMicaSenseLog">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_MicasenseTimer">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_MiaSenseStartOrNo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserMicaSenseLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>901</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_SavetoSD">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>SD卡:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MicaSenseLive3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Red LiveView</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_MicaSenseWebSetting">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>后台设置</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_LiveViewOrNo">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_MicaSenseLive6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Thermal LiveView</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MicasenseTimer">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_MiCaSenseCaptureSignal">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>信号触发</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LiveViewOrNo_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>型号选择:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CameraModel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RedEdge MX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Altum</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RedEdge M</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
